src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,51 @@
+package duke.task;
```

Comment:
Very impressive use of inheritance to create each Task subclass! It's neatly organised.